go.mod

```diff
@@ -539,7 +538,7 @@ replace (
 	k8s.io/kms => ./staging/src/k8s.io/kms
 	k8s.io/kube-aggregator => ./staging/src/k8s.io/kube-aggregator
 	k8s.io/kube-controller-manager => ./staging/src/k8s.io/kube-controller-manager
-	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20220803162953-67bda5d908f1
+	k8s.io/kube-openapi => github.com/benluddy/kube-openapi v0.0.0-20220929174216-3b907d108da1
```

Comment:
personal repo?

Reply:
Yes. This PR is only to demonstrate that in-tree openapi-gen output isn't changed by the related kube-openapi PR (not to actually be merged).

Reply:
/hold